libnmstate/nm/common.py

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# This file is part of nmstate
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+import gi
+
+try:
+    gi.require_version("NM", "1.0")  # NOQA: F402
+    from gi.repository import NM  # pylint: disable=no-name-in-module
+except ValueError:
+    NM = None
+
+from gi.repository import GLib
+from gi.repository import GObject
+from gi.repository import Gio
+
+__all__ = ["NM", "GLib", "GObject", "Gio"]
```

Comment:
This exposes these through `import *`. I would not recommend doing that, it encourages bad practice.

Reply:
1. It's internal use only.
2. Include `__all__` is suggested by pylint to get rid of `Import but not used error`.